source/blog/2017-06-22-introducing-buildah.html.md

```diff
@@ -0,0 +1,258 @@
+---
+title: Buildah - build your containers from the ground up!
+author: tsweeney
+date: 2017-06-22 15:00:00 UTC
+tags: containers, images, docker, buildah, oci
+comments: true
+published: true
+---
+
+Since I'm relatively new to the world of containers and images, I was excited to learn about [the Buildah tool](https://github.com/projectatomic/buildah).  Especially since I'm a native New Englander and it's a clever play on how we say Builder in these parts.
+
+Buildah is a newly released command line tool for efficiently and quickly building [Open Container Initiative](https://www.opencontainers.org/) (OCI) compliant images and containers.  Buildah simplifies the process of creating, building and updating images while decreasing the learning curve of the container environment.  It is easily scriptable and can be used in an environment where one needs to spin up containers automatically based on calls from your application.  What's really neat is there is no requirement for a container runtime daemon to be running on your system chewing up resources and complicating the build process.
+
+READMORE
```

Comment:
It creates a break in the post, between what's in the CSS feed and the rest of the article.